Libraries/LibChess/Chess.h

```diff
@@ -200,16 +180,21 @@ void Chess::generate_moves(Callback callback, Colour colour) const
 
         bool keep_going = true;
         if (piece.type == Type::Pawn) {
-            keep_going =
-                try_move({sq, {sq.rank+1, sq.file}}) &&
-                try_move({sq, {sq.rank+2, sq.file}}) &&
-                try_move({sq, {sq.rank-1, sq.file}}) &&
-                try_move({sq, {sq.rank-2, sq.file}}) &&
-                try_move({sq, {sq.rank+1, sq.file+1}}) &&
-                try_move({sq, {sq.rank+1, sq.file-1}}) &&
-                try_move({sq, {sq.rank-1, sq.file+1}}) &&
-                try_move({sq, {sq.rank-1, sq.file-1}});
+            for (auto& piece : Vector({ Type::None, Type::Knight, Type::Bishop, Type::Rook, Type::Queen })) {
+                // clang-format off
+                keep_going =
+                    try_move({sq, {sq.rank+1, sq.file}, piece}) &&
```

Comment:
Unless you really want `A &&` here, you can start the lines with `&&` and clang-format will keep them on different lines IIRC

Reply:
Thanks! that really annoyed me